include/seqan3/io/sam_file/detail/format_sam_base.hpp

```diff
@@ -481,9 +480,9 @@ inline void format_sam_base::read_header(stream_view_type && stream_view,
                         else
                             throw format_error{std::string{"Illegal SAM header tag: S"} + *it};
                     }
-                    else if (!is_char<'G'>(*it))                        // GO (grouping) tag
+                    else                                                // GO (grouping) tag, pb (PacBio) version, etc
                     {
-                        throw format_error{std::string{"Illegal SAM header tag in @HG starting with: "} + *it};
+                        seqan3::debug_stream << "Not supported SAM header tag in @HD starting with: " << *it << '\n';
```

Comment:
```suggestion
                        std::cerr << "Not supported SAM header tag in @HD starting with: " << *it << '\n';
```

? debug_stream is a bit overkill since it includes half the library and we don't need to format stuff here

Reply:
Ah, I did not know that. I thought that would be better than cerr, but no problem, I change that directly :) thanks for the tip